README.md

```diff
@@ -460,6 +472,16 @@ Expects the following parameters:
 
 Returns [`Promise<HTTP.Body>`](#body)
 
+#### <a name="getAvailable">`db.options.getAvailable(conn)`</a>
+
+Gets all possible database options with their default values.
```

Comment:
There appears to be mismatch in the wording between the description text here and what's declared in the index.d.ts file. These should be generated by running `yarn docs`.